requirements.txt

```diff
@@ -1,10 +1,134 @@
-awscli
-dsnap>=0.1.10
-boto3>=1.15
-botocore>=1.16
-s3transfer>=0.3.0
-requests==2.25.1
-urllib3==1.26.4
-SQLAlchemy==1.3.22
-SQLAlchemy-Utils==0.36.8
-typing-extensions==3.7.4.3
```

Comment:
This lives in pyproject.toml now, this file is generated from that with `poetry export`